src/Field.js

```diff
@@ -62,11 +62,11 @@ export default function Field(field) {
 		this.title = field.title;
 	}
 
-	if (field.min) {
+	if (field.min !== undefined) {
```

Comment:
I think `field.min/max` are always expected to be numbers, correct? If so, checking `if (typeof field.min === 'number')` might be a more robust check here. That said, I suppose they could also be numbers-as-strings, which would fail that check, so this is probs a-okay as well :+1:

Reply:
Yeah, `field.min/max` should always be numbers, good call! I updated the check since it doesn't seem like we support number-as-strings based on the actual `SirenField` implementation: https://github.com/Brightspace/D2L.Hypermedia.Siren/blob/master/D2L.Hypermedia.Siren/ISirenField.cs#L17-L19 and it's more consistent with the assert above

Reply:
Hah, you'd think I could have known that since I originally wrote it, but that would be expecting some level of "remembering literally anything" from me 😅

Reply:
You couldn't have known because I added it! 😅

Reply:
oh so it's your fault then

(jks, but also it was `long` before that)